powertools-e2e-tests/handlers/pom.xml

```diff
@@ -30,7 +30,8 @@
         <module>logging</module>
```

Comment:
You should merge down from v2 soon; there's a nadful of changes around the E2E stuff we pushed in with the #1402